content/docs/get-started/applying.md

```diff
@@ -37,17 +44,23 @@ running `mlem apply`:
 $ echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
 0,1,2,3" > new_data.csv
 
-$ mlem apply models/rf new_data.csv -i --it pandas[csv]
+$ mlem apply models/rf new_data.csv \
+    -m predict_proba -i --it "pandas[csv]"
 ⏳️ Importing object from new_data.csv
 ⏳️ Loading model from models/rf.mlem
-🍏 Applying `predict` method...
-[[0.3, 0.3, 0.4]]
+🍏 Applying `predict_proba` method...
+[[0.47 0.24 0.29]]
 ```
 
-`-i` and `--it pandas[csv]` tells MLEM it's a csv file that should be read with
-Pandas. For that to work, your data should be in a format that is supported by
-[MLEM import](/doc/user-guide/importing). You can learn more about specifying
-these arguments on `mlem apply` page.
+- The `--import`/`-i` flag tells MLEM to import the data on the fly.
+- The `--import-type` / `--it` flag, helps MLEM understand the data format.
+  Here, it's `pandas[csv]` a csv file that should be read with Pandas. For that
+  to work, your data should be in a format that is supported by
+  [MLEM import](/doc/user-guide/importing). You can learn more about specifying
+  these arguments on `mlem apply` page.
+
+- The `--method`/`-m` flag tells MLEM to invoke the `predict_proba` method and
+  return the class probabilities, instead of the default `predict`.
```

Comment:
This makes the GS more verbose. In the last iteration, we tried to remove as much details from it as possible - make it showcase a core value of the product and send to other parts of docs for details.

Citing @jorgeorpinel's [page](https://www.notion.so/iterative/Higher-level-content-9408f84a9b694fab949feede39142122):
> Get started: Races to demonstrate the product’s value via key features; Tends to use supporting materials

Should we simplify this? Or should we describe each argument used In a detailed fashion? cc @jorgeorpinel

Reply:
You're right
wdyt @aguschin ?
collapsible block? remove all flags explanation? link to some other section?
I just saw some flags detailed and didn't want to leave this half-explained

Reply:
Ok, pushed suggested changes.

Reply:
Thanks for following docs guidelines 🙂 sorry couldn't comment earlier. Seems this is resolved though.